metricbeat/module/system/filesystem/filesystem.go

```diff
@@ -32,6 +34,13 @@ func New(base mb.BaseMetricSet) (mb.MetricSet, error) {
 		return nil, err
 	}
 
+	if len(config.IgnoreTypes) == 0 {
```

Comment:
The behaviour I would have expected that if a user sets `filesystem.ignore_types: []` that then no file systems would be ignore. But checking for 0 indicates that it also applies in this case. Could we instead check if the config is set?

If we go with the above suggestion, the reference config file should be adjusted to contain the two values.

Reply:
Oh, right, checking nil instead of zero lenght. I have added a comment also in reference config.